src/features/authentication/login/store/LoginStore.ts

```diff
@@ -0,0 +1,45 @@
+import { action, makeObservable, observable } from "mobx";
+
+interface LoginData {
+  loading: boolean;
```

Comment:
I suggest naming variables with boolean type with "is", "are", "should" etc. as suffix to better code readability. In this case it would be `isLoading`. Here's a useful link with boolean naming conventions https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig.

Reply:
I have resolved this now and I shall review the link about best practices about naming variables. Thank you! :D